proto/system_config.proto

```diff
@@ -139,6 +142,9 @@ message SwitchSetup {
 
   // IP address template and subnet for module ip addresses
   string mods_addr = 16;
+ 
+  // Switch connect Telnet port
```

Comment:
I think this needs a slightly better name "telnet_port" is very generic... maybe cmd_port?  Talk about what it's use is, not the underlying protocol.